recipes/unfold/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "unfold" %}
+{% set version = "1.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d8a68f36bd58e036f920435d41fa8edc85a3643ec26b4507b3ca8eee6b491560
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.10
+    - setuptools
+  run:
+    - bw2data
+    - bw2io
+    - datapackage
+    - numpy
+    - pandas
+    - prettytable
+    - pyprind
+    - python >=3.10
+    - pyyaml
+    - scipy
+    - sparse
+    - wurst >=0.4.0
+
+test:
+  imports:
+    - unfold
+
+about:
+  home: https://github.com/simplejson/simplejson
```

Comment:
This url looks leftover from the sample. Please update it.

Reply:
Argh, that's true, just fixed it, thanks.